crypto/finite_field.ts

```diff
@@ -178,5 +178,22 @@ function createField(p: bigint, t: bigint, twoadicRoot: bigint) {
     fromBigint(x: bigint) {
       return mod(x, p);
     },
+    rot(x: bigint, bits: number, direction: boolean = true) {
```

Comment:
can we make `direction: 'left' | 'right'` so that it's clear what the parameter means without looking at the function code?

Reply:
done!